pkg/volume/fc/fc_util.go

```diff
@@ -361,9 +361,10 @@ func (util *FCUtil) DetachBlockFCDisk(c fcDiskUnmapper, mapPath, devicePath stri
 	}
 	loop, err := volumeutil.BlockVolumePathHandler.GetLoopDevice(blkUtil, dstPath)
 	if err != nil {
-		glog.Warningf("fc: failed to get loopback for device: %v, err: %v", dstPath, err)
-	} else {
-		glog.V(4).Infof("fc: found loopback: %v", loop)
+		if err.Error() != volutil.ErrDeviceNotFound {
```

Comment:
This should check `ErrNotAvailable` instead of `ErrDeviceNotFound`.

Reply:
ok. We should fix error code of GetLoopDevice().

Reply:
Done